### PR TITLE
New module to allow sales tax in reconciliation form

### PR DESCRIPTION
**Pull request by mdietrichc2c**
This new module allows to set sales taxes in the "Reconcile" form in Bank Statements.

### CODE REVIEW

**Comment by fclementic2c** [comment]
:+1:

File: account_statement_sale_tax/README.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Account Statement Sale Tax
+==============
```
Comment (pedrobaeza) [inline account_statement_sale_tax/README.rst]: Put same line length

File: account_statement_sale_tax/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline account_statement_sale_tax/__init__.py]: Please put short headers like https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L1-L3

File: account_statement_sale_tax/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Account Statement Sale Tax',
+    'version': '8.0.0.0',
```
Comment (pedrobaeza) [inline account_statement_sale_tax/__openerp__.py]: Should be 8.0.1.0.0

File: account_statement_sale_tax/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    'category': 'Generic Modules/Accounting',
```
Comment (pedrobaeza) [inline account_statement_sale_tax/__openerp__.py]: Only "Accounting"

**Comment by mdietrichc2c** [comment]
@pedrobaeza thanks for the review ; I corrected your points and rebased everything in a single commit.

**Comment by pedrobaeza** [comment]
Travis error is unrelated and concerns _account_asset_management_ (what I don't understand is why is now in red).

I tried on runbot and I confirm that works, but have you tried to pass this as a bug to Odoo?

**Comment by leemannd** [comment]
:+1:

**Comment by adrienpeiffer** [comment]
Hi,
Yesterday, I just did a PR to Odoo to fix that. see https://github.com/odoo/odoo/pull/11673.
Without precautions, the sign of the base and the tax is wrong with properly configured taxes.

**Comment by adrienpeiffer** [comment]
Hi,
It's now possible to select sales taxes in the reconciliation of bank statement in Odoo 8.0
See : https://github.com/odoo/odoo/commit/600f884524f7ff0ee747795eba644c0eb864559c

**Comment by sbidoul** [comment]
@adrienpeiffer cool, good news.

So this PR becomes unnecessary, right?

**Pull request closed without merging**